circuit/src/test_helpers.rs

```diff
@@ -0,0 +1,91 @@
+pub mod test_helpers {
```

Comment:
```suggestion
```
We already define the module in `lib.rs`, so no need for this

Reply:
this was me trying to put them all in one place